YIF.Core.Data/Entities/Lector.cs

```diff
@@ -3,7 +3,8 @@
 
 namespace YIF.Core.Data.Entities
 {
-    public class Lecture : BaseEntity
+    [Table("Lector")]
```

Comment:
why do you need this attribute?

Reply:
![image](https://user-images.githubusercontent.com/48133795/119980253-b5d40b80-bfc4-11eb-98c3-46e596b7a7fd.png)
and what about these lines in migration class?